deploy/0-RequestResponseConsumer.ts

```diff
@@ -8,11 +8,6 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
   console.log('0-RequestResponseConsumer.ts')
 
-  if (network.name != 'baobab') {
```

Comment:
After removing this block, `network` is imported but not used anywhere else. Please see the warning above.